ci-operator/step-registry/fips-check/node-scan/fips-check-node-scan-commands.sh

```diff
@@ -96,10 +96,9 @@ EOF
 else
     cat >"${ARTIFACT_DIR}/${filename}.xml" <<EOF
     <testsuite name="${testsuite}" failures="1" errors="0" skipped="0" tests="1" time="$SECONDS">
-        <testcase name="${subteam}:Node scan of fips check should succeedded or skipped"/>
+        <testcase name="${subteam}:Node scan of fips check should succeedded or skipped">
             <failure message="">
                 Node scan failed due to errors or warnings.
-                $res
```

Comment:
How about keeping it so that in RP you know it is error or warning at first glance?
```suggestion
                Node scan failed due to errors or warnings: $res.
```

Reply:
No, it is the special characters which make the junit report not readable. But I think it should be fixed with in the latest version.